cmd/notary/tuf.go

```diff
@@ -302,10 +299,10 @@ func tufPublish(cmd *cobra.Command, args []string) {
 	if err != nil {
 		fatalf(err.Error())
 	}
-	timestamp, err := filestore.GetMeta("timestamp", 0)
-	if err != nil {
-		fatalf(err.Error())
-	}
+	//timestamp, err := filestore.GetMeta("timestamp", 0)
```

Comment:
Yeah sorry. I habitually leave it in for my own reference for a few commits, but I'm aware it clutters things too. Pulling it now